contrib/babelfishpg_common/src/datetimeoffset.c

```diff
@@ -15,6 +15,7 @@
 #include "utils/timestamp.h"
```

Comment:
Are there sufficient test cases to prove the correct ness of this change?

Reply:
I have added extensive testcases in the past but the changes done here do not completely make sense to me. Sumit seems to have just copied what we did for the utc getdate functions like sysutcdatetime.

Reply:
No need for additional test cases, I have added them before

Reply:
can you point it? in which file?